packages/php-table-backend-utils/tests/Unit/Column/Bigquery/BigqueryColumnTest.php

```diff
@@ -66,6 +66,22 @@ public function testCreateFromDBNotNullInt(): void
         self::assertEquals('38,9', $column->getColumnDefinition()->getLength());
     }
 
+    public function testCreateFromNumericEmpty(): void
+    {
+        $data = [
+            'name' => 'age',
+            'type' => 'NUMERIC',
+            'mode' => 'REQUIRED',
+        ];
```

Comment:
tady to testuje default v RESTtoSQLDatatypeConverter co sem měnil okolo precision a scale